### ISSUE TITLE
Way to close Donkey (and underlying Vertx)

### ISSUE THREAD

**Issue by jarppe** [AppsFlyer/donkey#19]
**Is your feature request related to a problem? Please describe.**
During development using namespace reloading tools like integrant, there should be a way to release the resources created by `com.appsflyer.donkey.core/create-donkey`. Otherwise new instance of `Donkey` will be created each time namespaces are reloaded. As far as I can tell, these resources are not garbage collected.

**Describe the solution you'd like**
A function `close-donkey` that would release all resources created by `create-donkey`.

**Describe alternatives you've considered**
Creating `donkey` instance only once, and storing it in somewhere where namespace reloading would not affect it. Haven't figured any way to do this.

**Additional context**
None

**Comment by jarppe** [comment]
My current work-around is:
```clojure
(defn- close-donkey [^Donkey donkey]
  (-> donkey
      (.-config)
      (:vertx)
      (.close)))
```

**Comment by yaronel** [comment]
@jarppe it will be added in the next release.
Please note that `config` is not documented and is not part of the public API. You should not use it in your code as it may be removed in future versions without notice.
